pkg/client/access_control_records_object_set.go

```diff
@@ -1,5 +1,7 @@
 // Copyright 2020 Hewlett Packard Enterprise Development LP
 
+// create true
+
```

Comment:
Why are these comments in ever file?

Reply:
It was ftl debug print :) , will remove it . thanks